chainstate/src/detail/bootstrap.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::io::{BufRead, Write};
+
+use chainstate_storage::BlockchainStorageRead;
+use chainstate_types::{BlockIndex, PropertyQueryError};
+use common::{chain::Block, primitives::id::WithId};
+use serialization::{Decode, Encode};
+
+use crate::{BlockError, ChainstateConfig};
+
+use super::{orphan_blocks::OrphanBlocks, query::ChainstateQuery};
+
+const DEFAULT_MIN_IMPORT_BUFFER_SIZE: usize = 1 << 22; // 4 MB
```

Comment:
I feel like Im missing something, but in what case do I (as a user) want to change the minimum size? I probably can use the maximum size to reduce a memory consumption or stop the import process if a huge block is encountered unexpectedly. But what benefits of specifying the custom minimal size?

Reply:
In no case. This is only usable for tests (both unit and functional tests).